plugin/mouse/mouse.py

```diff
@@ -221,6 +228,13 @@ def mouse_gaze_scroll():
             actions.tracking.control_toggle(True)
             control_mouse_forced = True
 
+    def toggle_mouse_gaze_scroll():
```

Comment:
I would name this `mouse_gaze_scroll_toggle` to be consistent with the other mouse actions